meeteval/wer/matching/cy_levenshtein.pyx

```diff
@@ -111,32 +121,39 @@ def _validate_inputs(reference, hypothesis, reference_timing, hypothesis_timing)
             f'{len(hypothesis)} != {len(hypothesis_timing)}'
         )
 
-    reference_timing = np.array(reference_timing)
-    hypothesis_timing = np.array(hypothesis_timing)
+    # reference_timing = np.array(reference_timing)
+    # hypothesis_timing = np.array(hypothesis_timing)
 
-    assert len(reference) == 0 or reference_timing.shape == (len(reference), 2), (
-    reference_timing.shape, len(reference))
-    assert len(hypothesis) == 0 or hypothesis_timing.shape == (len(hypothesis), 2), (
-    hypothesis_timing.shape, len(hypothesis))
-    assert len(reference) == 0 or len(hypothesis) == 0 or reference_timing.dtype == hypothesis_timing.dtype, (
-    reference_timing.dtype, hypothesis_timing.dtype)
+    # assert len(reference) == 0 or reference_timing.shape == (len(reference), 2), (
+    # reference_timing.shape, len(reference))
+    # assert len(hypothesis) == 0 or hypothesis_timing.shape == (len(hypothesis), 2), (
+    # hypothesis_timing.shape, len(hypothesis))
+    # assert len(reference) == 0 or len(hypothesis) == 0 or reference_timing.dtype == hypothesis_timing.dtype, (
+    # reference_timing.dtype, hypothesis_timing.dtype)
 
-    def check(timing, info):
-        # end >= start
-        if np.any(timing[:, 1] < timing[:, 0]):
+    cdef double last_start = 0
+    for t in reference_timing:
+        if t.second < t.first:
             raise ValueError(
-                f'The end time of an interval must not be smaller than its begin time, but the {info} violates this')
-        # start values are increasing
-        if np.any(np.diff(timing[:, 0]) < 0):
+                f'The end time of an interval must not be smaller than its begin time, but the reference violates this')
```

Comment:
Add information, that the error msg contains more information as the code, e.g. `f'The end time {t.second} of an interval must not be smaller than its begin time {t.first}, but the reference violates this'`

Reply:
Done